dev/cnf/build.gradle

```diff
@@ -252,73 +252,29 @@ task createGeneratedReplacementProjects {
 
 def releaserepo = file(bnd('releaserepo', 'release')) /* Release repository. */
 
-task ('createIndex') {
-    dependsOn copyMavenLibs
-    dependsOn everythingElseHasBeenReleased
-    description 'Index the release repository.'
-    group 'release'
+import aQute.bnd.gradle.Index
 
+task ('index', type: Index) {
+    description 'Index the release repository. (Does not depend on releaseNeeded)'
+    group 'release'
+    repositoryName = "Bnd ${version}"
```

Comment:
You probably want a more Open Liberty name since you are indexing OL bundles and not Bnd jars.